ROMFS/px4fmu_common/init.d/airframes/13000_generic_vtol_standard

```diff
@@ -24,6 +24,27 @@
 
 . ${R}etc/init.d/rc.vtol_defaults
 
+param set-default CA_AIRFRAME 2
+param set-default CA_ROTOR_COUNT 4
+param set-default CA_ROTOR0_PX 0.15
```

Comment:
As we anyway normalize the effectiveness, we can just as well use 1 instead of 0.15 for the quads, that may make it clearer that it's normalized.

Reply:
I wanted to make sure it's in the right region, so users don't think the have to change it if close to their setup, and should we ever want to make use of it, we can.

Reply:
Alright that's fine as well ofc.